src/main/java/bio/terra/cli/service/GoogleOauth.java

```diff
@@ -371,6 +379,16 @@ public void onTokenErrorResponse(Credential credential, TokenErrorResponse token
     }
   }
 
+  /** Get the client secrets by server. */
+  private static GoogleClientSecrets UpdateClientSecret() {
```

Comment:
just implement this in getClientSecret. have readClientSecrets takes in clientSecretFileName